src/commands.rs

```diff
@@ -763,6 +763,9 @@ where
 /// And longer paths precede shorter paths
 /// And if a path is a sub-path of the other, then only paths with shallower hierarchy (closer to the top-level) remain
 fn prepare_paths_for_deletion(paths: &mut Vec<Vec<String>>) {
+    if paths.len() < 2 {
```

Comment:
Can you explain this addition? Is this a bug?

Reply:
Minor efficiency improvement (not a bug - just avoid the overhead of re-ordering when less than 2 paths exist)

Reply:
So lets add a comment in the code explaining this optimization.